components/brave_rewards/resources/ui/components/modalVerify/index.tsx

```diff
@@ -71,7 +71,9 @@ export default class ModalVerify extends React.PureComponent<Props, {}> {
 
   getFooter = () => (
     <StyledFooter>
-      <span dangerouslySetInnerHTML={{ __html: getLocale('walletVerificationFooter') }} />
+      <span>
+        {getLocale('walletVerificationFooter')}
```

Comment:
not sure where to test this, maybe go to brave://rewards and then click verify wallet?